From: Dmitry Shachnev <mitya57@debian.org>
Date: Wed, 23 May 2018 15:56:02 +0300
Subject: Add *.md files to the MANIFEST

Without this, 'setup.py build' does not copy them to the build
directory, and some tests are failing.
---
 MANIFEST.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index 0a3afb3..b730196 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,5 +1,5 @@
 include README.md
 include LICENSE
-recursive-include mkdocs *.ico *.js *.css *.png *.html *.eot *.svg *.ttf *.woff *.woff2 *.xml *.mustache *mkdocs_theme.yml
+recursive-include mkdocs *.ico *.js *.css *.png *.html *.eot *.svg *.ttf *.woff *.woff2 *.xml *.mustache *mkdocs_theme.yml *.md
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
